### ISSUE TITLE
Sentry fix with Laravel 5.5

### ISSUE THREAD

**Issue by vojtasvoboda** [skydiver/october-plugin-sentryerrorlogger#5]
Just letting you know that I made a fix to Sentry library: https://github.com/getsentry/sentry-laravel/pull/107

It's recommended to use:

```
"require": {
    "sentry/sentry-laravel": "dev-master#4166a9b"
}
```

But it is up to you if you want to fix it now or wait for version 0.9.0.

**Comment by skydiver** [comment]
Hello @vojtasvoboda 
It's ok for me, wan't to do the PR?

Thanks.-

**Comment by vojtasvoboda** [comment]
Fixed by #6
